### ISSUE TITLE
Image::getExtension() is throwing a deprecation error in PHP8.1 due to missing return type

### ISSUE THREAD

**Issue by alexmart** [eymengunay/php-passbook#97]
When trying to generate a passbook file with `eo/passbook:v3.1.1` on `PHP8.1` I get this error:

```Deprecated: Return type of Passbook\Pass\Image::getExtension() should either be compatible with SplFileInfo::getExtension(): string, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice {"exception":"[object] (ErrorException(code: 0): Deprecated: Return type of Passbook\\Pass\\Image::getExtension() should either be compatible with SplFileInfo::getExtension(): string, or the #[\\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice at /www/vendor/eo/passbook/src/Passbook/Pass/Image.php:97)"} []```

Just adding the return type string to Image::getExtension() method seems to fix this.
```diff
--- src/Passbook/Pass/Image.php
+++ src/Passbook/Pass/Image.php
@@ -94,7 +94,7 @@
     /**
      * {@inheritdoc}
      */
-    public function getExtension()
+    public function getExtension(): string
     {
         if ($this->forceExtension) {
             return 'png';
```

Should I create a PR with this fix?

**Comment by eymengunay** [comment]
Hello @alexmart, 

Thanks for letting me know! Due time constraints I'm not actively contributing to this repository anymore so I'd really appreciate a PR!

**Comment by bluewire-solutions** [comment]
issue confirmed

**Comment by endsoonconfess** [comment]
Hello @eymengunay,

Please take a few minutes to review PR with solution to remove deprecation notices emitted on actual versions of PHP

Thank you!

**Comment by eymengunay** [comment]
This PR https://github.com/eymengunay/php-passbook/pull/99 which addresses the issue has been merged. Thank you @endsoonconfess .

**Comment by endsoonconfess** [comment]
Thank you too, 
my team is migrating codebase to 8.2 and I would really appreciate new tag to pull updated lib into project

**Comment by razvanphp** [comment]
Released as [v3.1.2](https://github.com/eymengunay/php-passbook/releases/tag/v3.1.2)
